sklearn/model_selection/tests/test_plot.py

```diff
@@ -116,7 +116,7 @@ def test_validation_curve_display_default_usage(pyplot, data):
         estimator, X, y, param_name=param_name, param_range=param_range
     )
 
-    assert display.param_range == param_range
+    assert_array_equal(display.param_range, param_range)
```

Comment:
This is exactly what the line below is doing indeed :)

Reply:
You can remove it then.

Reply:
Oups! :sweat_smile:
```suggestion
```